### ISSUE TITLE
Nano Ledger Error 32602

### ISSUE THREAD

**Issue by Flazky** [paritytech/parity#7820]
> I'm running:
>
> - **Which Parity version?**: 1.9.2
> - **Which operating system?**: Windows 10
> - **How installed?**: via installer 
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum 
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

1)Parity client detects a connected Nano Ledger S, but is unable to confirm request throwing error "32602 invalid params: invalid type: null,  expected string" 

No prompt on the Nano Ledger S itself either.

2)Parity does not show Accounts list if Nano Ledger S is connected before starting the Parity Client.

**Comment by rbutler81** [comment]
I may be having the same issue... posted here:  https://ethereum.stackexchange.com/questions/39046/parity-1-9-2-ledger-nano

**Comment by Flazky** [comment]
It is most likely an issue with the new Parity client itself. I have tried using the nano with MEW and works fine.

**Comment by svantetobias** [comment]
I have the same problem in Parity 1.9.4. @5chdn you gave a hint on reddit that this should have been fixed in 1.9.4. Know anything about it? :)

**Comment by jamespic** [comment]
The issue looks to be that `signer_confirmRequest` is being called with `null` as a password, which the backend doesn't accept. If I inject an empty string as the password, then I can sign transactions. I can't tell whether it's going to be easier to make the backend accept null, or the front-end send send a non-null password (I don't totally grok the signer code yet).

**Comment by m5x** [comment]
I have the same issue with Parity 1.9.5 and Trezor. Is there some workaround? I really need to send some ethereum out...

**Comment by jamespic** [comment]
You can work around it with `parity signer list`, `parity signer sign SIGNING_REQUEST_ID` from the command line.

**Comment by m5x** [comment]
Thanks for the hint. Worked like a charm.

**Comment by toasteruk** [comment]
I have the same issue. It's way beyond my skills to use the command line (wherever that is). What do I do?

**Comment by jamespic** [comment]
@toasteruk I don't know of an easier workaround than that.

If you're on Windows or Ubuntu, you can get a command line by clicking the start menu, and typing "command line" (not sure about Mac). On Ubuntu or Mac the commands should work as-is, on Windows you may need to change to the directory you installed Parity to.

Another workaround would be to use MyEtherWallet, if it works for your use case.

**Comment by toasteruk** [comment]
I was having issues with a dapp that I was using my ledger with. Parity is
the only option until Metamask add ledger support. I got it sorted though
by reverting to an older version of Parity. Thanks

**Comment by jamespic** [comment]
Funnily enough, this issue (and the more general context of the Parity UI being deprecated) prompted me to look at adding Ledger support to MetaMask. I made some progress (which I discussed on https://github.com/MetaMask/metamask-extension/issues/717), but some weird architectural choices by Chrome are making it much harder than it needs to be (either Chrome or MetaMask would need to make some painful changes to their security model). It might actually be easier to write a new Chrome plugin that's Ledger-only, but I've never quite found the motivation (or time) to do so.

**Comment by toasteruk** [comment]
Oh right! Well I can certainly say you'd be a hero to many of us Ledger
owners if did find the time :)

**Comment by Xiaoyang-Zhu** [comment]
I got the same issue on version 1.9.6? any fix patches?

**Comment by cwesystems** [comment]
For me Trezor now works with "Parity 1.10.1-beta". Maybe you can give that version a try?

**Comment by shane1900** [comment]
Same error for me even after trashing app and installing latest stable release on mac.

**Comment by jamespic** [comment]
@shane1900 What about 1.10.1, which is the latest *beta* release?

**Comment by Tbaut** [comment]
@shane1900 Could you please try with Parity v1.10 and to connect the Ledger Nano S before launching Parity and Parity UI ?

**Comment by shane1900** [comment]
@Tbaut Yes that works with v1.10 thank you.  I was not wanting to use the beta since previously it had not worked with Maker / Dai CDP's.  Another bug though, setting custom gas did not take on the first screen.  Had to edit it manually again on the confirmation popup.

**Comment by Tbaut** [comment]
Thanks @shane1900, can you confirm you're on windows?

**Comment by shane1900** [comment]
Yes v.1.10 on windows @Tbaut
